app/pages/analyst/application-review.tsx

```diff
@@ -99,16 +113,48 @@ class ApplicationReview extends Component<Props> {
             <IncentiveCalculatorContainer
               applicationRevision={query.applicationRevision}
             />
-          </Col>
-          <Col md={4} className="review-step-box">
-            <ApplicationReviewStep reviewStep="Technical" />
-          </Col>
+            <OverlayTrigger
+              placement="top"
+              overlay={(props) => (
+                <Tooltip id="scroll-to-top" {...props}>
+                  Return to top of page
+                </Tooltip>
+              )}
+            >
+              <Button
+                id="to-top"
+                variant="dark"
+                type="button"
+                onClick={() => {
+                  const main = document.body.getElementsByTagName('main')[0];
+                  main?.scrollIntoView();
+                }}
+                style={{
+                  borderRadius: '50%',
+                  position: 'fixed',
+                  bottom: '0.6rem',
+                  left: '0.6rem',
+                  width: 50,
+                  height: 50,
+                  boxShadow:
+                    '0 4px 5px 0 rgba(0, 0, 0, 0.14), 0 1px 10px 0 rgba(0, 0, 0, 0.12), 0 2px 4px -1px rgba(0, 0, 0, 0.4)'
+                }}
+              >
+                <FontAwesomeIcon size="lg" icon={faArrowUp} />
+              </Button>
+            </OverlayTrigger>
+          </div>
+          {this.state.isSidebarOpened && (
+            <ReviewSidebar
+              applicationReviewStep={
+                query.application.applicationReviewStepsByApplicationId.edges[0]
```

Comment:
This is where we set the currently selected review step to display in the sidebar. In future this will be set based on what they've selected in the review step menu, but for now it displays the first review step retrieved.

All existing applications' comments have been migrated in #1538 to a unique review step named "Legacy" review, and there are no comments yet referencing the new review steps - so I think that it's okay to temporarily set the displayed step in this way until the selector menu is implemented.